shared/msg_conversions/src/msg_conversions.cc

```diff
@@ -18,90 +18,97 @@
 
 #include <msg_conversions/msg_conversions.h>
 
+#ifdef ROS1
 #include <ros/console.h>
+#else
+// Setup logging for ROS2
+#include "rclcpp/rclcpp.hpp"
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("msg_conversions");
```

Comment:
Following the pattern of making the code look like the ROS2 API (with a standard prefix), ideally we would want callers to write this:
```c++
auto logger = ff_ros::rclcpp::get_logger("msg_conversions");
if (crashed) {
  FF_ROS_RCLCPP_FATAL_STREAM(logger, ... stuff to log ...);
}
```
A sketch of how to support that:
```c++
namespace ff_ros {
#if ROS1
  namespace rclcpp {
    typedef Logger bool;
    inline Logger get_logger(const std::string& name) { return true; }
  }
#else
  namespace rclcpp = ::rclcpp;
#endif
}

#if ROS1
#define FF_ROS_RCLCPP_FATAL_STREAM(LOGGER, ...) ROS_FATAL_STREAM(__VA_ARGS__)
#else
#define FF_ROS_RCLCPP_FATAL_STREAM(LOGGER, ...) RCLCPP_FATAL_STREAM(LOGGER, __VA_ARGS__)
#endif
```